### DISCUSSION TITLE
Plotter looking for non-existant file

### DISCUSSION THREAD

**Discussion by Zurginator** [Chia-Network/chia-blockchain#6005] [Support]
I checked the drives for these files and nothing close:

What L drive hasfiles in the following stage:
plot-k32-2021-05-24-06-04-e1c2cfeefb6665da993e3d08d87a52e097bb1f5578f3a0d88bd1997901f2b9a7.plot.p3s.t4.sort_bucket_010.tmp

Errors:
Could not open L:\plot-k32-2021-05-24-06-04-e1c2cfeefb6665da993e3d08d87a52e097bb1f5578f3a0d88bd1997901f2b9a7.plot.p3.t4.sort_bucket_014.tmp: No such file or directory. Retrying in five minutes.
Could not open K:\plot-k32-2021-05-24-06-04-99ab2739b2e79d863cc1d46231e4dd8307a7291dca233c45800696f68ed23d5b.plot.p3s.t4.sort_bucket_052.tmp: No such file or directory. Retrying in five minutes.
Could not open M:\plot-k32-2021-05-24-06-07-8f865762fba32fd6c32b09b1e8156aa93e68138a49f49a4b2bc48f8072e70a6a.plot.p3.t4.sort_bucket_022.tmp: No such file or directory. Retrying in five minutes.**

**Comment by BrandtH22** [comment]
@Zurginator , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - make sure that the item listed in the Could not open message is not listed in the plot directories of the gui.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
